### DISCUSSION TITLE
请问author-year还能像numerical那样保留参考文献的序号吗

### DISCUSSION THREAD

**Discussion by Juelianqvq** [stone-zeng/fduthesis#250] [Q&A] [answered]
请问author-year还能像numerical那样保留参考文献的序号，如[1],[2]这样吗？
![屏幕截图 2022-03-23 185211](https://user-images.githubusercontent.com/42963620/159683368-dfea7fe8-e2c0-427e-9ac8-ed6b576dab00.png)

**Comment by stone-zeng** [comment, marked as answer]
需要将选项 `style/bib-backend` 设置为 `biblatex`，同时在导言区加上

```tex
\PassOptionsToPackage{
  sorting = gb7714-2015,
  gblanorder = chineseahead,
  sortlocale = zh__pinyin,
}{biblatex}
```

注意此时需要用 `biber` 而非 `bibtex` 生成参考文献。具体细节请参考 `biblatex-gb7714-2015` 宏包文档的 2.4 小节。

**Reply by stone-zeng** [reply]
需要说明的是，这里的方案实际上是给 `numerical` 样式进行了额外的排序，而不是给 `author-year` 样式增加序号。引用时 `author-year` 会采用 `(著者名, 年份)` 的格式，这时对参考文献表编号会很奇怪。

**Reply by Juelianqvq** [reply]
您好，请问biblatex-gb7714-2015 宏包文档有链接吗？抱歉没能找到...

**Reply by syvshc** [reply]
> 您好，请问biblatex-gb7714-2015 宏包文档有链接吗？抱歉没能找到...

命令行运行 `texdoc biblatex-gb7714-2015`

**Reply by stone-zeng** [reply]
[biblatex-gb7714-2015.pdf](https://mirrors.ctan.org/macros/latex/contrib/biblatex-contrib/biblatex-gb7714-2015/biblatex-gb7714-2015.pdf)
